lib/account/pages/about_page.dart

```diff
@@ -1,5 +1,11 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:url_launcher/url_launcher_string.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+const String githubLink = 'https://github.com/MOTIS-Mitfahr-App/flutter-app';
+const String websiteLink = 'https://motis-project.de/';
```

Comment:
i know we don't have a terms of service link but can we add that just for completeness and use the same as websiteLink for now.